src/api/app/models/status_message.rb

```diff
@@ -29,6 +29,16 @@ def delete
 
   def acknowledge!
     users << User.session!
+  rescue ActiveRecord::RecordInvalid => e
+    return false if users.include?(User.session!)
```

Comment:
This could be `return false if acknowledged?`

Reply:
Yes